### PR TITLE
Added traceback to help print the the exception and what led to it.

### PR DESCRIPTION
**Pull request by cgisala**
@4LineTime I added a traceback to the to help with the troubleshooting of what caused the issue.

### CODE REVIEW

**Comment by cgisala** [comment]
@4LineTime changed the message in the except to give a more user friendly message.

File: main.py [hunk 1]
```diff
@@ -19,7 +19,8 @@ def main():
             if choice == 'Q':
                 break
         except:
-            traceback.print_last()
+            print("\nYou have this book already.\n")
```
Comment (claraj) [inline main.py:22]: This is not appropriate here. If your program experiences any error your user will see a 'You have this book already' message which is probably not what the problem is. Each function should handle errors as part of that function

**Comment by claraj** [comment]
How does it help a user to see a traceback? For this problem you needed to deal with the cause of the error. Users typically don't understand tracebacks.  If there's a problem with code, print a user-friendly error message. Logging is more appropriate for storing messages for developers.

**Comment by claraj** [comment]
The title of the PR is too vague. Which issue is this fixing? What error? what exception?  Be specific about what problem this is fixing.

**Comment by claraj** [comment]
A better solution is a try-except around  this line of code. If you don't get a BookError raised, display the added book message. If there is a BookError, display an error message "you already have  this book" https://github.com/4LineTime/readinglist/blob/bbff186a5e09b6c91eb5f795771650486327cef9/main.py#L44

**Pull request merged**